### ISSUE TITLE
matomo:4.15.1 is missing in Docker Hub for linux/amd64

### ISSUE THREAD

**Issue by iamspido** [matomo-org/docker#330]
matomo:4.15.1 is missing in Docker Hub for linux/amd64.
Is there something wrong in the tagging?
matomo:4.15.1-fpm has all 6 OS/ARCH but matomo:4.15.1 has only 3 OS/ARCH.
Same is for 4.15.1-apache.

**Comment by nightcoding** [comment]
+1 for this. Tried upgrading today to `4.15.1` and image for linux/amd64 still not available. I found an issue in the main repository with a link to the following forum post: https://forum.matomo.org/t/docker-images-for-4-15-1-are-incomplete-in-os-support/52641

Spoiler: no solution given until yet. I think we have to wait :)

**Comment by michalkleiner** [comment]
@iamspido @nightcoding is this image (https://hub.docker.com/layers/library/matomo/latest/images/sha256-a2834605ca7aea8473c5bb67d045cb0780ee462a36cea5e2361f555e72f1b8c0?context=explore) not working?

**Comment by jurgenhaas** [comment]
I'm having the same issue and not, the FPM image is not what I need. I need the apache image which has always been available up until 4.15.0, but not for 4.15.1

**Comment by iamspido** [comment]
@michalkleiner We need the apache version of matomo. matomo:4.15.1 and matomo:4.15.1-apache are the versions with apache.

**Comment by michalkleiner** [comment]
Right, I see it now and yeah, it's missing from Docker Hub. Unfortunately I don't have any control of the build system, so we'd need to ping @J0WI and wait.

@J0WI I'd be happy to be added as a maintainer/contributor wherever needed in order to be able to kick the rebuilds off etc. Happy to help.

**Comment by iamspido** [comment]
linux/amd64 is now available.

**Comment by michalkleiner** [comment]
@J0WI I'm one of the 'people from here', I was more thinking in terms of the Jenkins builds and anything beyond the repo itself and access to Docker hub to push the images up (if that is even how it's done).
